src/lib.rs

```diff
@@ -0,0 +1,17 @@
+#![feature(array_chunks)]
+#[macro_use]
+extern crate log;
+
+/// Minecraft implementations.
+pub mod minecraft;
+/// Mojang API implementation.
+pub mod mojang;
+
+pub use minecraft::{client::MinecraftClient, server::MinecraftServer, status::StatusChecker};
+pub use mojang::auth;
+
+/// Initialize the library. Required before using any components
+pub fn init() {
+    env_logger::init();
```

Comment:
This should be run in the main, function, it does not need a wrapper. (There should however be a doc comment in the top of this lib.rs file stating that env_logger is used, it is up to the library user to initialize env_logger how they want)